lib/parser/interfaces.ts

```diff
@@ -70,7 +70,7 @@ export function implementsIHasContextReference(obj: O.ObjectBase): obj is O.Obje
   return (obj as O.ObjectBase & Partial<IHasContextReference>).contextReferences !== undefined;
 }
 export interface IHasReferenceLinks {
-  referenceLinks: O.OReference[];
+  referenceLinks: O.OName[];
```

Comment:
```suggestion
  nameLinks: O.OName[];
```
Don't know about that...